dists/go-ipfs/mkdist.js

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env node
```

Comment:
:-1: I'm strongly against these shebangs with node, I have seen to many setups where this simply fail or do not do what people expect them to.